src/main/java/com/scmp/framework/context/FrameworkConfigs.java

```diff
@@ -119,4 +119,19 @@ public FrameworkConfigs(ConfigurableApplicationContext context, Environment env)
 
 	@Value("${EXTENT_XML_PATH:#{null}}")
 	private String extentXMLPath;
+
+	@Value("${NOTIFICATION_SEND_FAILED_CASE:#{false}}")
+	private boolean sendFailedCaseNotification;
```

Comment:
@benliscmp can you please help update the ReadMe as well?

Reply:
Updated readme.md